<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataPenyewa</class>
 <widget class="QDialog" name="DataPenyewa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/BG2.jpg);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/white.jpg);
color: rgb(0, 0, 0);
image: url(:/res/white.jpg);</string>
   </property>
   <property name="text">
    <string>Pengemudi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="durasiLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nikLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pengemudiLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/orange.jpg);
image: url(:/res/orange.jpg);</string>
   </property>
   <property name="text">
    <string>Urutkan</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>451</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);
border-top-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/white.jpg);
color: rgb(0, 0, 0);
image: url(:/res/white.jpg);</string>
   </property>
   <property name="text">
    <string>Nama</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namaLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/white.jpg);
color: rgb(0, 0, 0);
image: url(:/res/white.jpg);</string>
   </property>
   <property name="text">
    <string>Durasi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/orange.jpg);
image: url(:/res/orange.jpg);</string>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/white.jpg);
color: rgb(0, 0, 0);
image: url(:/res/white.jpg);</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/orange.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/white.jpg);
color: rgb(0, 0, 0);
image: url(:/res/white.jpg);</string>
   </property>
   <property name="text">
    <string>Mobil</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/white.jpg);
font: 12pt &quot;Rockwell&quot;;
color: rgb(0, 0, 0);
image: url(:/res/white.jpg);</string>
   </property>
   <property name="text">
    <string>Data Penyewa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/orange.jpg);
image: url(:/res/orange.jpg);</string>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/orange.jpg);
image: url(:/res/orange.jpg);</string>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
